CHANGELOG.md

```diff
@@ -7,11 +7,16 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
-## [0.1.0] - 2023-02-07
+### Added
+
+- Allow cluster domain customization on servicemonitors and modules.
+- Add json schema.
 
-## [0.0.1] - 2023-02-07
+### Changed
 
-## [0.0.1] - 2023-02-07
+- Default interval to 10 seconds on all probes.
```

Comment:
when I tested this it didn't work at all

Reply:
```
# Source: prometheus-blackbox-exporter/templates/servicemonitor.yaml
apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
metadata:
  name: blackbox-exporter-prometheus-blackbox-exporter-dns-udp-internal
  namespace: kube-system
  labels:
    helm.sh/chart: prometheus-blackbox-exporter-1.0.0
    app.kubernetes.io/name: prometheus-blackbox-exporter
    app.kubernetes.io/instance: blackbox-exporter
    app.kubernetes.io/version: "0.23.0"
    app.kubernetes.io/managed-by: Helm
    application.giantswarm.io/team: "phoenix"
    application.giantswarm.io/team: phoenix
spec:
  endpoints:
  - port: http
    scheme: http
    path: "/probe"
    interval: 10s
    scrapeTimeout: 30s
```

I see it on the servicemonitor output, did you also delete it from the specific probes?